### ISSUE TITLE
Comparison failure in text_axes:test_phase_spectrum_freqs

### ISSUE THREAD

**Issue by mdboom** [matplotlib/matplotlib#5525]
In #5307, where I am lowering our testing tolerance down to zero, I am discovering a few recent changes that slipped through because the tolerance on our tests was too high.

This test has been broken since at least v1.4.0 (before which I can't build with current Numpys).

BASELINE RESULT:

![phase_spectrum_freqs-expected](https://cloud.githubusercontent.com/assets/38294/11286923/15cd582c-8ee6-11e5-887c-8e099aa62913.png)

ACTUAL RESULT:

![phase_spectrum_freqs](https://cloud.githubusercontent.com/assets/38294/11286927/1c8f18a8-8ee6-11e5-9128-6fc099e91344.png)

Test code:

```
@image_comparison(baseline_images=['phase_spectrum_freqs'],
                  remove_text=True,
                  extensions=['png'])
def test_phase_spectrum_freqs():
    '''test axes.phase_spectrum with sinusoidal stimuli'''
    n = 10000
    Fs = 100.

    fstims1 = [Fs/4, Fs/5, Fs/11]

    NFFT = int(1000 * Fs / min(fstims1))
    pad_to = int(2 ** np.ceil(np.log2(NFFT)))

    x = np.arange(0, n, 1/Fs)

    y = np.zeros(x.size)
    for i, fstim1 in enumerate(fstims1):
        y += np.sin(fstim1 * x * np.pi * 2) * 10**i
    y = y

    fig = plt.figure()
    ax1 = fig.add_subplot(3, 1, 1)
    ax2 = fig.add_subplot(3, 1, 2)
    ax3 = fig.add_subplot(3, 1, 3)

    spec1, freqs1, line1 = ax1.phase_spectrum(y, Fs=Fs, pad_to=pad_to,
                                              sides='default')
    spec2, freqs2, line2 = ax2.phase_spectrum(y, Fs=Fs, pad_to=pad_to,
                                              sides='onesided')
    spec3, freqs3, line3 = ax3.phase_spectrum(y, Fs=Fs, pad_to=pad_to,
                                              sides='twosided')

    ax1.set_xlabel('')
    ax2.set_xlabel('')
    ax3.set_xlabel('')
    ax1.set_ylabel('')
    ax2.set_ylabel('')
    ax3.set_ylabel('')
```

@toddrjen: You appear to have written the test.  Any thoughts?

**Comment by dopplershift** [comment]
Looks to me like the original result had the phase wrap around pi, and that's not in the current result. Hard to tell without ticks though,

**Comment by mdboom** [comment]
Just an update: I thought this might be related to a change in numpy, since that's where most of the actual calculation happens.  However, going back to 1.7 (the earliest that matplotlib head compiles with), the result is all the same.

I'm leaning on just updating the baseline image to reflect the new result if I don't hear otherwise.

**Comment by dopplershift** [comment]
No argument here. Minus the wrap (which is going to depend on implementation details of the calculation), the images do appear to match still.

**Comment by tacaswell** [comment]
:+1: to updating the images.  This is probably an API change in mlab, but no one has complained about that yet so we should probably let it go.

**Comment by QuLogic** [comment]
I tried out c89ce08c767eff13a1c83d6b5c9f5db2bae702c2 where the test image was added (against NumPy 1.8.2), but it is also incorrect there. In fact, I went back a little further to 7fc90de7387bd7511c520f485aeb50d162f554aa where the `phase_spectrum` plotting method was added and it also does not produce the baseline image.

**Comment by QuLogic** [comment]
It also fails against NumPy 1.6.2, since you thought it might be related.

**Comment by mdboom** [comment]
Ok.  All signs point to just updating the baseline image on this one then.  Thanks for the extra investigation.  I'll probably just update the image as part of #5307.

**Comment by mdboom** [comment]
Closed by #5307.
